api/v3/clairpb/clair.proto

```diff
@@ -155,16 +155,11 @@ message GetAncestryResponse {
     message Ancestry {
         // The name of the desired ancestry.
         string name = 1;
-        // The detectors used to scan this Ancestry. It may not be the current
-        // set of detectors in clair status.
-        repeated Detector detectors = 2;
         // The list of layers along with detected features in each.
         repeated AncestryLayer layers = 3;
     }
     // The ancestry requested.
     Ancestry ancestry = 1;
-    // The status of Clair at the time of the request.
-    ClairStatus status = 2;
```

Comment:
We still need the status so that clients can determine when they need to rescan something.

Reply:
:+1: